frontend/src/metabase-types/api/mocks/query.ts

```diff
@@ -8,6 +8,7 @@ import type {
 export const createMockStructuredQuery = (
   opts?: Partial<StructuredQuery>,
 ): StructuredQuery => ({
+  "source-table": 1,
```

Comment:
```suggestion
  "source-table": PRODUCTS_ID,
```

plus

```tsx
import { PRODUCTS_ID } from "metabase-types/api/mocks/presets";
```

Reply:
I think we want presets to depend on mock object helpers, but not vice-versa